docs/import/manage_disaster.js

```diff
@@ -675,41 +683,87 @@ function toggleState(known) {
   }
 }
 
+const scoreAssetTypes = [
+  ['poverty', ['snap_data', 'paths'], 'Poverty'],
+  ['income', ['income_asset_paths'], 'Income'],
+  ['svi', ['svi_asset_paths'], 'SVI'],
+];
+Object.freeze(scoreAssetTypes);
+
+const assetSelectionRowPrefix = 'asset-selection-row-';
+Object.freeze(assetSelectionRowPrefix);
+
+/**
+ * Initializes score selector table based on {@link scoreAssetTypes} data. Done
+ * as soon as page is ready.
+ */
+function setUpScoreSelectorTable() {
+  const tbody = $('#asset-selection-table-body');
+  for (const scoreAssetType of scoreAssetTypes) {
+    const row = $(document.createElement('tr'));
+    row.append(createTd().text(scoreAssetType[2]));
+    row.prop('id', assetSelectionRowPrefix + scoreAssetType[0]);
+    tbody.append(row);
+  }
+}
+
 /**
  * Initializes the select interface for score assets.
  * @param {Array<string>} states array of state (abbreviations)
  */
 function initializeScoreSelectors(states) {
   const headerRow = $('#score-asset-header-row');
-  const tableBody = $('#asset-selection-table-body');
-  tableBody.empty();
-  headerRow.empty();
 
   // Initialize headers.
-  headerRow.append(createTd().html('Score Assets'));
+  removeAllButFirstFromRow(headerRow);
   for (const state of states) {
     headerRow.append(createTd().html(state + ' Assets'));
   }
 
   // For each asset type, add select for all assets for each state.
   for (const scoreAssetType of scoreAssetTypes) {
-    const row =
-        $(document.createElement('tr')).prop('id', scoreAssetType + '-row');
-    row.append(createTd().append(
-        $(document.createElement('div')).html(scoreAssetType)));
+    const id = assetSelectionRowPrefix + scoreAssetType[0];
+    const propertyPath = scoreAssetType[1];
+    const row = $('#' + id);
+    removeAllButFirstFromRow(row);
     for (const state of states) {
       if (stateAssets.get(state)) {
+        const pathDictionary = getElementFromPath(propertyPath);
         const select =
-            createAssetDropdown(stateAssets.get(state), scoreAssetType, state);
+            createAssetDropdown(stateAssets.get(state), pathDictionary[state]);
+        select.val(pathDictionary[state]);
```

Comment:
maybe make constant?

Reply:
select.val shouldn't still be there, removed.